learn/what_is_meilisearch/overview.mdx

```diff
@@ -17,18 +17,24 @@ _Meilisearch helps the Rust community find crates on [crates.meilisearch.com](ht
 
 ## Features
 
-- **Blazing fast** (answers < 50 milliseconds): Priority is given to fast answers for a smooth search experience.
-- [Search as you type](/learn/what_is_meilisearch/features#search-as-you-type): Results are updated on each keystroke. To make this possible, we use [prefix-search](/learn/advanced/prefix#prefix-search).
-- [Typo tolerance](/learn/what_is_meilisearch/features#typo-tolerant): Understands typos and misspellings.
-- [Tokenization](/learn/advanced/tokenization) in **English**, **Chinese**, and **all languages that use space as a word divider**.
-- **Return the whole document**: The entire document is returned upon search.
+- **Blazing fast**: Answers in less than 50 milliseconds
+- **Search as you type**: Results are updated on each keystroke using [prefix-search](/learn/advanced/prefix#prefix-search)
+- [Typo tolerance](/learn/configuration/typo_tolerance): Get relevant matches even when queries contain typos and misspellings
+- [Comprehensive language support](/learn/what_is_meilisearch/language): Optimized support for **Chinese, Japanese, Hebrew, and languages using the Latin alphabet**
+- **Returns the whole document**: The entire document is returned upon search
 - **Highly customizable search and indexing**:
```

Comment:
Huh, just noticed we don't have anything here, which clashes hard against every other feature having some descriptive text. We should add something so it's in line with the other items.